src/uct/ib/mlx5/dc/dc_mlx5_ep.h

```diff
@@ -292,19 +304,18 @@ uct_dc_mlx5_ep_rand_arb_group(uct_dc_mlx5_iface_t *iface, uct_dc_mlx5_ep_t *ep)
 static UCS_F_ALWAYS_INLINE ucs_arbiter_group_t*
 uct_dc_mlx5_ep_arb_group(uct_dc_mlx5_iface_t *iface, uct_dc_mlx5_ep_t *ep)
 {
-    return (uct_dc_mlx5_iface_is_policy_shared(iface)) ?
-            uct_dc_mlx5_ep_rand_arb_group(iface, ep) : &ep->arb_group;
+    return uct_dc_mlx5_iface_is_policy_shared(iface) ?
+                   uct_dc_mlx5_ep_rand_arb_group(iface, ep) :
```

Comment:
maybe rename uct_dc_mlx5_ep_rand_arb_group to uct_dc_mlx5_ep_shared_arb_group ?
even though that function is not changed in this PR, seems like a better name for it